projects/effortless.md

```diff
@@ -0,0 +1,47 @@
+# Effortless
+
+The Effortless project includes the Habitat Scaffolding for Chef Infra and Chef InSpec and the things to make using them easy.
```

Comment:
I'm not 100% sure this is the right wording we want to use to describe `Effortless`, is it a project? or a combination of patterns?

Reply:
Workflow?